### DISCUSSION TITLE
Skip to content

### DISCUSSION THREAD

**Discussion by starconfig** [bootscore/bootscore#864] [General]
Hi

I'm wondering why we have on every page this element?
`<a class="skip-link visually-hidden-focusable" href="#primary">Skip to content</a>`

**Comment by crftwrk** [comment]
The `Skip to content` link is used for accessibility for people who using keyboard, screen readers or other assistive technologies. 

See https://www.w3.org/TR/WCAG20-TECHS/G1.html
